js/core/telemetryV3Interface.js

```diff
@@ -556,21 +556,27 @@ var Telemetry = (function() {
             'canvas': true,
             'screenResolution': true,
             'availableScreenResolution': true,
-            'touchSupport': true,
-            'plugins': true,
-            'webgl': true,
-            'audio': true,
-            'language': true,
-            'deviceMemory': true
+            // 'touchSupport': true,
+            // 'plugins': true,
+            // 'webgl': true,
+            // 'audio': true,
+            // 'language': true,
+            // 'deviceMemory': true
         },
         NOT_AVAILABLE: 'not available',
         ERROR: 'error',
         EXCLUDED: 'excluded'
     }
     this.telemetry.getFingerPrint = function (cb) {
-        Fingerprint2.getV18(FPoptions, function (result, components) {
-            if (cb) cb(result, components)
-        })
+        if (localStorage.getItem('fp_deviceId')) {
+            const {result,components} = JSON.parse(localStorage.getItem('fp_deviceId'));
```

Comment:
use var and assign value to deviceDetails

Reply:
done